post_office/models.py

```diff
@@ -198,7 +199,7 @@ def prepare_email_message(self, connection=None):
                         att.is_inline = True
                         att.content_id = match.group(1)
                 for attachment in self.attachments.all():
-                    path = os.path.join(settings.MEDIA_ROOT, attachment.file.name)  # should be the relative path
+                    path = default_storage.save(os.path.join(settings.MEDIA_ROOT, attachment.file.name), attachment.file.name)  # should be the relative path
```

Comment:
We don't need to save a copy in the default storage. We already have a copy there. Either this needs to be changed to add the content the file to be added as an attachment (preferred option), or written to disk somewhere locally temporarily and then that path is used. The issue with writing to disk is how the email is sent. If it is sent as a cronjob, then that is not a viable solution